app/assets/stylesheets/dm/pages/_home.scss

```diff
@@ -4,27 +4,10 @@
 
 .dm-homepage {
   .homepage-welcome-banner {
-    .homepage-welcome-banner-body-text-container {
-      margin-bottom: 119px;
-
-      @media screen and (min-width: 1024px) {
-        @include u-margin-bottom(0);
-      }
-    }
-  }
-
-  .homepage-welcome-banner-image-container {
-    @include u-position('absolute');
-    bottom: -87px;
-
-    @media screen and (min-width: 1024px) {
-      height: 223px;
-      @include u-right(0);
-      bottom: unset;
-    }
+    background: #162E51;
```

Comment:
Nice cleanup on this section. This matches the footer color, which uses `.bg-primary-darker`. Let's use the utility class on the HTML element